ingest/entryReader.go

```diff
@@ -337,6 +344,39 @@ headerLoop:
 					er.ackChan <- ackCommand{cmd: CONFIRM_TAG_MAGIC, val: uint64(tg)}
 				}
 			}
+			continue
+		case INGESTER_STATE_MAGIC:
+			// read length of buffer
+			n, err = io.ReadFull(er.bIO, er.buff[0:4])
+			if err != nil {
+				return err
+			}
+			if n < 4 {
+				return errFailedFullRead
+			}
+			length := binary.LittleEndian.Uint32(er.buff[0:4])
```

Comment:
https://tools.ietf.org/html/draft-newman-network-byte-order-01

network byte order is big endian for what it's worth.